packages/outputarea/package.json

```diff
@@ -54,7 +54,7 @@
     "@lumino/properties": "^1.8.1",
     "@lumino/signaling": "^1.10.1",
     "@lumino/widgets": "^1.30.0",
-    "resize-observer-polyfill": "^1.5.1"
+    "resize-observer": "^1.0.4"
```

Comment:
Switching to another polyfill based on the discussions in https://github.com/que-etc/resize-observer-polyfill/issues/83

Reply:
We could probably drop the polyfill, no?

https://caniuse.com/?search=ResizeObserver

Reply:
Yes that would be great.

I left it for now just to check if we have any commitment related to the minimal browser versions, and which browsers we support.

Reply:
The docs say: https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html?#supported-browsers

![image](https://user-images.githubusercontent.com/591645/144472651-c0e55582-0c47-40f5-acfe-8dc210f9cc59.png)

So we could indeed just drop the polyfill.